### PR TITLE
add unibit 🎉

### PR DESCRIPTION
**Pull request by SaraVieira**
Adds template for unibit

Adds nunjucks extension

Screenshot 2019-07-08 at 12 15 45

https://github.com/codesandbox/codesandbox-importers/pull/48

### CODE REVIEW

File: packages/common/src/templates/icons.ts [hunk 1]
```diff
@@ -118,6 +119,8 @@ export default function getIcon(theme: TemplateType): ReturnedIcon {
       return MDXDeckIcon;
     case quasar.name:
       return QuasarIcon;
+    case unibit.name:
+      return QuasarIcon;
```
Comment (SaraVieira) [inline packages/common/src/templates/icons.ts]: waiting for icon

File: packages/app/package.json [hunk 2]
```diff
@@ -130,7 +130,7 @@
     "@babel/preset-env": "^7.4.2",
     "@cerebral/http": "^4.0.0",
     "@cerebral/mobx-state-tree": "^3.0.0",
-    "@codesandbox/template-icons": "^0.3.0",
+    "@codesandbox/template-icons": "^0.4.0",
```
Comment (Saeris) [inline packages/app/package.json]: Latest version has the Unibit icon

```suggestion
    "@codesandbox/template-icons": "^0.5.0",
```

File: packages/common/src/templates/icons.ts [hunk 3]
```diff
@@ -24,6 +24,7 @@ import {
   SapperIcon,
   ParcelIcon,
   VuePressIcon,
+  StackbitIcon,
```
Comment (Saeris) [inline packages/common/src/templates/icons.ts]: Swap out for the Unibit Icon

```suggestion
  UnibitIcon,
```

File: packages/common/src/templates/icons.ts [hunk 4]
```diff
@@ -118,6 +120,8 @@ export default function getIcon(theme: TemplateType): ReturnedIcon {
       return MDXDeckIcon;
     case quasar.name:
       return QuasarIcon;
+    case unibit.name:
+      return StackbitIcon;
```
Comment (Saeris) [inline packages/common/src/templates/icons.ts]: Update Icon Reference

```suggestion
      return UnibitIcon;
```

**Pull request merged**